src/metabase/util/honey_sql_2.clj

```diff
@@ -32,7 +32,7 @@
   (sqlize [this]
     (let [numerator   (.numerator ^clojure.lang.Ratio this)
           denominator (.denominator ^clojure.lang.Ratio this)]
-      (clojure.core/format "(%d / %d)" numerator denominator))))
+      (clojure.core/format "(%d.0 / %d.0)" numerator denominator))))
```

Comment:
`select 1/2` in h2 returns 0 so we get divide by zero errors.

Reply:
that makes sense, maybe we should just change the inline behavior for Ratio to call `double` on it